packages/main/src/MenuItem.ts

```diff
@@ -202,6 +205,33 @@ class MenuItem extends ListItem implements IMenuItem {
 	@i18n("@ui5/webcomponents")
 	static i18nBundle: I18nBundle;
 
+	_itemNavigation: ItemNavigation;
+
+	constructor() {
+		super();
+
+		this._itemNavigation = new ItemNavigation(this, {
+			navigationMode: NavigationMode.Horizontal,
+			behavior: ItemNavigationBehavior.Static,
+			getItemsCallback: () => this._navigableItems,
+		});
+	}
+
+	get _navigableItems() {
+		return [...this.endContent] as Array<HTMLElement>;
+	}
```

Comment:
It will be better to have it as a return type of the getter.
```
get _navigableItems() : Array<HTMLElement> {
		return [...this.endContent];
	}
```

Reply:
Done